### PR TITLE
Solo review

### PR DESCRIPTION
**Pull request by assadounto**
### Summary
This is a project from the microverse trials by Rich and queeniva which will be reviewed by Richmond

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# Projectreview
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (assadounto) [inline README.md]: Here Readme should be well documented and about the project

File: index.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <h1 id="title" class="title">Software Survey Form</h1>
+  <form id="survey-form">
+    <div class="name">
+      <h3>Name</h3>
```
Comment (assadounto) [inline index.html:15]: Name should be of input type name

**Review by assadounto — COMMENTED** [review]
Please make the necessary correction as it will make your code more readable

File: index.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```
Comment (gtekle) [inline index.html:11]: - [ ] You should have mentioned that the best practice for javascript using `<script>` tags is by adding them just before the closing tag of the `body` element for faster page load.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  <h1 id="title" class="title">Software Survey Form</h1>
+  <form id="survey-form">
+    <div class="name">
+      <h3>Name</h3>
+      <label id="name-label" </label>
+      <input id="name" placeholder="Enter Your Name">
+    </div>
+    <div class="email">
+          <h3>Email</h3>
+      <label id="email-label"</label>
```
Comment (gtekle) [inline index.html:21]: - [ ] You should have mentioned that the `label` element is not working and the `<h3>` tag is redundant & not semantically meaningful to use it as a form element.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
```
Comment (gtekle) [inline index.html:10]: - [ ] You should have mentioned that the body is not divided into semantically meaningful section for SEO and accessibility reasons. For example, `header, section, main, footer, etc`

**Review by gtekle — APPROVED** [review]
Hi @assadounto :wave:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: comments are polite
:heavy_check_mark: comments are clear

## Optional suggestions

- [ Optional ] It is good that you gave comment to add a professional readme. However, it would have great if you could `the reason why` we need a professional documentation and suggest that a [template](https://github.com/microverseinc/readme-template/blob/master/README.md) like the one provided by Microverse is used as a guidline.

- [Optional ] The summary of your comments 'apply all the necessary changes' seems a bit open and general. You could have said `please check the inline comments and you should apply the changes.`

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**